gef-ui/server/controllers/application-details/index.test.js

```diff
@@ -131,6 +131,7 @@ describe('controllers/application-details', () => {
           unissuedFacilitiesPresent: expect.any(Boolean),
           facilitiesChangedToIssued: expect.any(Array),
           displayComments: expect.any(Boolean),
+          hasChangedFacilities: expect.any(Boolean),
```

Comment:
Any chance we could add a better test in here for the facilities? At the moment it just expects anything - changes have been made and test for this is passing. Could we add `mapSummaryList` into the facilities part of the test?

Reply:
changed (without mapSummaryList)